src/styles/util/variable.scss

```diff
@@ -0,0 +1,7 @@
+$backimg: url(/src/images/star.svg);
+$activestar: url(/src/images/star-active.svg);
+$blue-accent: #00ACDC;
+$main: #060B35;
+$secondary: #616070;
```

Comment:
try to keep a certain standard when creating names so that it is clear what kind of constant it is

for example, what is secondary? write secondary-color instead
better to write like this
```
 $main-color: #060b35;
 $secondary-color: #616070;
 $blue-color: #00acdc;
 $white-color: #fff;
 ```
 also write colors in lowercase like #00acdc instead of #00ACDC